edx_proctoring/templates/proctoring/seq_proctored_exam_entrance.html

```diff
@@ -38,44 +38,81 @@
+      <i class="fa fa-arrow-circle-right start-timed-exam" data-attempt-proctored=false></i>
   </button>
 </div>
+<div class="proctored-exam-skip-confirm-wrapper hidden">
+  <div class="proctored-exam-skip-confirm">
+    <div class="msg-title">
+      {% blocktrans %}
+        Are you sure you want to take this exam without proctoring?
+      {% endblocktrans %}
+    </div>
+    <div class="msg-content">
+      {% blocktrans %}
+        You will no longer be eligible to use this course for academic credit.
+      {% endblocktrans %}
+    </div>
+    <div class="proctored-exam-skip-actions">
+      <button class="proctored-exam-skip-cancel-button action action-secondary">
+        {% trans "Cancel" %}
+      </button>
+      <button class="proctored-exam-skip-confirm-button action action-primary" data-ajax-url="{{enter_exam_endpoint}}" data-exam-id="{{exam_id}}" data-attempt-proctored=false data-start-immediately=true>
+        {% trans "Yes, Take this as an open exam" %}
+      </button>
+    </div>
+    <div class="clearfix"></div>
+    </div>
+</div>
 {% include 'proctoring/seq_proctored_exam_footer.html' %}
 
 <script type="text/javascript">
+  var startProctoredExam = function(exam_id, action_url, start_immediately, attempt_proctored) {
+    $.post(
+      action_url,
+      {
+        "exam_id": exam_id,
+        "attempt_proctored": attempt_proctored,
+        "start_clock": start_immediately
+      },
+      function(data) {
+        // reload the page, because we've unlocked it
+        location.reload();
+      }
+    );
+  };
   $('.start-timed-exam').click(
     function(event) {
-      var action_url = $(this).data('ajax-url');
-      var exam_id = $(this).data('exam-id');
       var attempt_proctored = $(this).data('attempt-proctored');
-      var start_immediately = $(this).data('start-immediately');
-
       if (!attempt_proctored) {
-        var msg = gettext(
-          "Are you sure you want to take this exam without proctoring? " +
-          "You will no longer be eligible to use this course for academic credit."
-        );
-        if (!confirm(msg)) {
-          return;
+        $(".proctored-exam.entrance").addClass('hidden');
+        $(".proctored-exam-skip-confirm-wrapper").removeClass('hidden');
+      }
+      else {
+        var action_url = $(this).data('ajax-url');
+        var exam_id = $(this).data('exam-id');
+        var start_immediately = $(this).data('start-immediately');
+        if (typeof action_url === "undefined" ) {
+          return false;
         }
+        startProctoredExam(exam_id, action_url, start_immediately, attempt_proctored);
       }
+    }
+  );
+  $('.proctored-exam-skip-confirm-button').click(
+    function(event){
+      var attempt_proctored = $(this).data('attempt-proctored');
+      var action_url = $(this).data('ajax-url');
+      var exam_id = $(this).data('exam-id');
+      var start_immediately = $(this).data('start-immediately');
```

Comment:
For this particular event (time exam confirmation), it is always going to be true, so no need to set/get this value from the element.